packages/compas-convert/src/passes/init-typescript-in-project.ts

```diff
@@ -49,10 +49,10 @@ export async function initTypescriptInProject(context: Context) {
 
 export default defineConfig({
   test: {
-  	globalSetup: "src/test/config.ts",
+    setupFiles: "./test/config.ts",
```

Comment:
Setup files are loaded in the correct context, which allows es module live binding sharing.